package.json

```diff
@@ -23,32 +23,27 @@
   "devDependencies": {
     "babar": "0.0.3",
     "cca": "~0.3.1",
-    "chalk": "~0.5.1",
     "crypto": "0.0.3",
     "freedom-for-chrome": "~0.4.7",
     "freedom-for-firefox": "~0.6.4",
     "freedom-pgp-e2e": "^0.2.3",
-    "fs-extra": "~0.12.0",
     "grunt": "~0.4.4",
     "grunt-browserify": "~3.0.1",
     "grunt-contrib-clean": "~0.6.0",
     "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-jasmine": "~0.8.2",
+    "grunt-jasmine-chromeapp": "^1.3.0",
```

Comment:
Probably worth giving this a go with 1.5.0, and I'm guessing there'll be a 1.6.0 or similar in the next day or so.